azure-pipelines.yml

```diff
@@ -118,25 +118,26 @@ jobs:
   displayName: Build Software for Earl Grey toplevel design
   dependsOn: lint
   condition: and(succeeded(), eq(dependencies.lint.outputs['DetermineBuildType.onlyDocChanges'], '0'))
-  pool: ci-public
+  pool:
+    vmImage: ubuntu-18.04
```

Comment:
just to double check here, have you experimented with this change? @GregAC is this OK?

Reply:
I might be able to make the job fit with the old pool. I'll stage a copy of this PR to better show the effect and discuss.

Reply:
@GregAC 's on holiday till Monday; addressing #13005 might give a path to moot this issue.